### PR TITLE
Milestone 2: Withdrawals

### PR DESCRIPTION
**Pull request by PeterBenc**
image

### CODE REVIEW

**Comment by refi93** [comment]
btw I guess it may be worth pinging Metamask devs that they forgot to translate "Approve" into Czech :)

**Review by refi93 — APPROVED** [review]
overall LGTM, just one question probably worth addressing

File: packages/cardano-metamask-snap/src/api/sdk/withdrawal.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    return [];
+  }
+
+  return Array.from(withdrawals.keys() ?? []).map((rewardAccount) => {
+    const amount = withdrawals.get(rewardAccount)?.toString();
+    assert(amount, 'Reward amount must be defined');
+
+    const rewardAccountKeyHashBech32 = keyHashHexToBech32(
+      Cardano.RewardAccount.toHash(Cardano.RewardAccount(rewardAccount)),
+      'stake_vkey',
```
[the remark this answers is not in the document]
Comment (PeterBenc) [reply packages/cardano-metamask-snap/src/api/sdk/withdrawal.ts]: Fixed :+1:

**Comment by PeterBenc** [comment]
Merged into develop

**Pull request closed without merging**